apps/storybook/src/overlays/AlertDialog.stories.tsx

```diff
@@ -0,0 +1,132 @@
+    title: "Delete Image",
+  },
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+    await userEvent.click(canvas.getByRole("button"));
+    await expect(
+      await screen.findByRole("alertdialog", { name: "Delete Image" }),
+    ).toBeInTheDocument();
+    await expect(
+      screen.getByRole("button", { name: "Confirm" }),
+    ).toBeInTheDocument();
+    await userEvent.click(screen.getByRole("button", { name: "Cancel" }));
+    await expect(
+      screen.queryByRole("alertdialog", { name: "Delete Image" }),
+    ).not.toBeInTheDocument();
+  },
+  render: Template,
+};
+
+export const CustomButtons: Story = {
+  args: {
+    ...Default.args,
+    action: "Yes, Delete",
+    cancel: "No, Keep",
+  },
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+    await userEvent.click(canvas.getByRole("button"));
+    await expect(
+      await screen.findByRole("alertdialog", { name: "Delete Image" }),
+    ).toBeInTheDocument();
+    await expect(
+      screen.getByRole("button", { name: "Yes, Delete" }),
+    ).toBeInTheDocument();
+    await expect(
+      screen.getByRole("button", { name: "No, Keep" }),
+    ).toBeInTheDocument();
+    await userEvent.click(screen.getByRole("button", { name: "No, Keep" }));
+    await expect(
+      screen.queryByRole("alertdialog", { name: "Delete Image" }),
+    ).not.toBeInTheDocument();
+  },
+  render: Template,
+};
+
+export const LongContent: Story = {
+  args: {
+    ...Default.args,
+    children:
+      "This is a longer piece of content that demonstrates how the AlertDialog handles more text. It might wrap to multiple lines depending on the width of the dialog.This is a longer piece of content that demonstrates how the AlertDialog handles more text. It might wrap to multiple lines depending on the width of the dialog.This is a longer piece of content that demonstrates how the AlertDialog handles more text. It might wrap to multiple lines depending on the width of the dialog.This is a longer piece of content that demonstrates how the AlertDialog handles more text. It might wrap to multiple lines depending on the width of the dialog.This is a longer piece of content that demonstrates how the AlertDialog handles more text. It might wrap to multiple lines depending on the width of the dialog.This is a longer piece of content that demonstrates how the AlertDialog handles more text. It might wrap to multiple lines depending on the width of the dialog.This is a longer piece of content that demonstrates how the AlertDialog handles more text. It might wrap to multiple lines depending on the width of the dialog.This is a longer piece of content that demonstrates how the AlertDialog handles more text. It might wrap to multiple lines depending on the width of the dialog.This is a longer piece of content that demonstrates how the AlertDialog handles more text. It might wrap to multiple lines depending on the width of the dialog.",
```

Comment:
can we make this longer? on my larger screen all the content fits and so i don't get a scrollbar